packages/loopring_v3/contracts/amm/libamm/AmmStatus.sol

```diff
@@ -71,16 +71,26 @@ library AmmStatus
             }));
         }
 
-        // Mint all liquidity tokens to the pool account on L2
-        S.balanceOf[address(this)] = AmmData.POOL_TOKEN_MINTED_SUPPLY;
-        S.allowance[address(this)][address(exchange.getDepositContract())] = type(uint256).max;
-        exchange.deposit(
-            address(this), // from
-            address(this), // to
-            address(this), // token
-            uint96(AmmData.POOL_TOKEN_MINTED_SUPPLY),
-            new bytes(0)
-        );
+        bool newPool = (S.accountID == 0);
```

Comment:
when/where is this `S.accountID` set to 0?

Reply:
It's only set in the first call to `setupPool` here, it's value can never be changed even if the pool gets reused.